### DISCUSSION TITLE
Link for Linux download doesn't work

### DISCUSSION THREAD

**Discussion by qwerty123t** [developersharif/bgremover-app#6] [General]
But I managed to download after deleteding "0" at the end of the link

**Comment by developersharif** [comment]
is it works now?

**Comment by qwerty123t** [comment]
Link in the Readme.md table for Linux still doesn't works

https://github.com/developersharif/bgremover-app/releases/download/v1.0.0/BG.Remover0


But this link works:
https://github.com/developersharif/bgremover-app/releases/download/v1.0.0/BG.Remover

**Reply by developersharif** [reply]
Thanks, link fixed!
